src/Bicep.Cli/Commands/PublishCommand.cs

```diff
@@ -78,19 +78,16 @@ public async Task<int> RunAsync(PublishArguments args)
             }
```

Comment:
Please manually test publishing with source using your changes and reading the sources using an older version of the bicep extension.

Reply:
Manually tested publishing with this version and restoring using an older version, and vice versa.